### DISCUSSION TITLE
Add Polyglot plugin for multilingual support

### DISCUSSION THREAD

**Discussion by ampuEus** [cotes2020/jekyll-theme-chirpy#2154] [Ideas]
### Description

I am implementing Polyglot to my own chirpy blog. And I was wondering if you think it can be useful to make a pull request to implement it here as well. **There are different ways to implement Polyglot and I think it's best for the community to decide how I integrate Polyglot with chirpy**. 

I think it is something that is being requested:

- #461 - How to let user choose between languages ?
- #604 - Chirpy theme multilingual
- #660 - Multiple Language
- #1818 - Add multilanguage support -> A discontinued implementation

And there are already people who have it working on their own:

- <https://www.yunseo.kim/>
- <https://aturret.space/>

So if you like, I will create a pull request to deal with the issue.

**Comment by ampuEus** [comment]
As this proposal doesn't have enough interest I'm don't going to implement it.

**Reply by LuizMeier** [reply]
I do have interest on that.

I've been using aturret's tutorials for some weeks in order to get polyglot working on my [blog](https://blog.lmeier.net).

Actually I found this thread looking for s solution for a problem that I have: my Polyglot is working perfectly except for the home page of the blog. I put a translation button at the sidebar and it works normally. However, the content of the home page does not get localization  working and I don't know why.

As I'm not a native dev (but I try do to my best), it would be great to have a native plugin doing the localization.

My code is [here](https://github.com/LuizMeier/blog).

**Comment by geekifan** [comment]
Hello, due to Jekyll's design, it doesn't natively support multilingual translation. I've migrated it to Hugo. Thanks to Hugo's architecture, it offers native multilingual support. Currently, 90% of the functionality has been implemented. If you'd like to try it out or contribute to the development, you can check out https://github.com/geekifan/hugo-theme-chirpy.

**Reply by lotusk08** [reply]
I'm really excited to try it!
